projects/ad796x_fmcz/builds.json

```diff
@@ -0,0 +1,10 @@
+{
+  "xilinx": {
+    "iio": {
+      "flags": "IIO_EXAMPLE=y BASIC_EXAMPLE=n",
+      "hardware": [
+        "ad796x_fmc_zed"
```

Comment:
the build is probably failing because of this... the hdl project is here from whay I have been tols: https://github.com/analogdevicesinc/hdl/tree/main/projects/pulsar_lvds_adc
so the hardware artifact is actually named pulsar_lvds_adc_zed
let's give this a try and see if the build is successful

Reply:
i tested using prebuilt a HDL shared sea-misc. i dont think this was the pulsar_lvds
as the readme in there does not mention anything about pulsar. Also the defines in parameters
do not not mention PULSAR. 

I think the HDL for this part still needs to be merged. i will ask